### PR TITLE
feat: automatic AccessToken refresh

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: src/main/auth/AccessTokenStore.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  public init(authAPI: AuthAPI, initialAccessToken?: AccessTokenData): Promise<AccessTokenData> {
+    this.authAPI = authAPI;
+
+    return this.tokenStore
+      .init(this.ACCESS_TOKEN_TABLE)
+      .then(() => {
+        if (initialAccessToken) {
+          return initialAccessToken;
+        } else {
+          return this.refreshAccessToken();
```
Comment (Yserz) [inline src/main/auth/AccessTokenStore.ts]: refreshed accessToken is never set

Comment (bennyn) [inline src/main/auth/AccessTokenStore.ts]: Fixed :)

Comment (bennyn) [inline src/main/auth/AccessTokenStore.ts]: `refreshAccessToken` now calls `updateToken` :)

**Comment by Yserz** [comment]
👍 Nice work!

**Comment by bennyn** [comment]
Thank you very much! 😃

**Pull request merged**